packages/cli/src/commands/optimize/optimize.js

```diff
@@ -0,0 +1,73 @@
+/**
+ * @flow
+ */
+
+const fs = require('fs').promises;
+const path = require('path');
+import sharp from 'sharp';
+import type {ConfigT} from 'types';
+import optimizeCommandLineArgs from './optimizeCommandLineArgs';
+import {logger} from '@react-native-community/cli-tools';
+
+function isImage(filePath: string) {
+  const regex = /\.(png|jpg|jpeg)/;
+  return filePath.toLowerCase().match(regex);
+}
+
+async function optimize(args: Array<string>, ctx: ConfigT) {
+  logger.log('Optimizing assets...');
+  const rootPathForImages = path.join(ctx.root, 'src');
```

Comment:
Having `src` is quite opinionated. People may keep images in `<projectRoot>/assets` dir for example. Maybe we should require a directory, or ask users if they didn't pass any?

Reply:
implementing *include* option, so users can pass their custom directory names